app/views/cart/show.html.erb

```diff
@@ -46,7 +46,15 @@
 </table>
   <p>Total: <%= number_to_currency(cart.total) %></p>
   <p><%= link_to "Empty Cart", "/cart", method: :delete %></p>
-  <p><%= link_to "Checkout", "/orders/new", method: :get %></p>
+  <% if current_user.role == 0%>
```

Comment:
Should we use enum? `current_user.visitor?` looks nicer than `current_user.role == 0`

Reply:
For our next run we can add that in